### DISCUSSION TITLE
Integration with qtranslate-XT

### DISCUSSION THREAD

**Discussion by northword** [pods-framework/pods#6823] [Ideas]
### Problem to Solve

I use the pods plugin to create custom post types and custom fields, and use [qtranslate-xt](https://github.com/qtranslate/qtranslate-xt) for multi-language, but qtranslate-xt can only take effect on the title and default editor, not on the custom fields created by Pods.

> qTranslate-XT (eXTended) - reviving qTranslate-X. A new community-driven plugin soon. GUTENBERG initial support now available! Built-in modules for WooCommerce, ACF, All in one SEO, Events Made Easy, Gravity Forms.
> https://github.com/qtranslate/qtranslate-xt

### Proposed Solution

Integration with qtranslate-XT, such as [acf-qtranslate](https://github.com/qtranslate/qtranslate-xt/tree/master/modules/acf). 

### Screenshots / Screencast

![](https://user-images.githubusercontent.com/44738481/163137777-72d5aaea-80d8-45c8-9f58-ccd478e9ef2d.png)
Fig. title and default editor have LSB

![](https://user-images.githubusercontent.com/44738481/163137790-2d1bb98c-a703-44cd-9867-6fdb91ac8d9f.png)
Fig. Fig. custom fields(毕业去向, text area) no LSB

**Comment by JoryHogeveen** [comment]
Hi @northword 

This plugin works vastly different than other plugins like WPML and Polylang who simply create new posts. These plugins work with Pods due to their structure.
Because qTranslate is built so differently it will require a dedicated addon to build this. I'm not sure where an addon like this would be in the scope of Pods features in the future.

Regards, Jory
